API/Models/UserDTO.cs

```diff
@@ -1,28 +1,8 @@
-﻿using System;
-using Newtonsoft.Json;
-using System.ComponentModel.DataAnnotations;
+﻿using Microsoft.AspNetCore.Identity;
 
 namespace API.Models
 {
-    public class UserDTO
+    public class UserDTO : IdentityUser
     {
-        [Key] 
-        [JsonProperty("id")] 
-        public int ID { get; set; }
-
-        [Required] 
-        [JsonProperty("email")] 
-        public string Email { get; set; }
-
-        [JsonProperty("username")]
-        public string Username { get; set; }
-
-        [Required]
-        [JsonProperty("password")] 
-        public string Password { get; set; }
-
-        [Required]
-        [JsonProperty("registered_at")]
-        public DateTime RegisteredAt { get; set; }
     }
```

Comment:
If this class just inherits `IdentityUser`, is it really necessarry to have it?

Reply:
Yes, this is the class where we'll have to add new properties when we need additional fields for the User table. The existing ones were removed because they are present by default.